frontend/src/components/auth/Profile.jsx

```diff
@@ -6,47 +6,150 @@ import { useState, useEffect } from 'react'
 import { useAuth } from './auth.jsx'
 import { useNavigate } from 'react-router-dom'
 
+
+const interviewQuestions = [
+    "Can you tell me about a time you worked on a team to complete a project? What was your role, and what did you learn from the experience?",
```

Comment:
let's move these into a text file? to make it easy to add/remove questions or upload a new set of questions (if for ex we wanted to have multiple versions for different experience levels in the future we could just change the name of the text file input rather than a whole new array

Reply:
Will make either a json or txt file for the interview questions.